app/views/imports/_form_main.html.erb

```diff
@@ -12,31 +12,30 @@
         </tr>
       </thead>
       <tbody>
-        <% if imports.first.persisted? %>
```

Comment:
Just changed the indent here - I'm not entirely clear what this check was doing?

Reply:
i wanna say this had something to do with the case of an empty `imports` table besides the "new" one, which is defined in the controller so it means `imports` will never be "empty"? so i guess it's in essence an "empty" check?

Reply:
That was my guess as well.  At some point since then, the "new" import was moved to the `@import` variable, and not appended to `@imports` table.